FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m

```diff
@@ -327,20 +327,26 @@ - (void)URLSession:(NSURLSession *)session
 }
 
 - (void)receiveWebSocketData {
+    __weak __autotype *weakSelf = self;
```

Comment:
I think there is a typo here and it should be `__weak __auto_type weakSelf = self;`. I can post a commit if it's already late for you.